packages/app/package.json

```diff
@@ -145,7 +145,7 @@
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/plugin-transform-runtime": "^7.12.10",
     "@babel/preset-typescript": "^7.12.7",
-    "@graphql-codegen/cli": "^1.20.1",
+    "@graphql-codegen/cli": "^2.2.0",
```

Comment:
Breaking change is deprecation of node 10, which does not affect us.
https://github.com/dotansimha/graphql-code-generator/blob/master/packages/graphql-codegen-cli/CHANGELOG.md#200